app/mailers/user_mailer.rb

```diff
@@ -197,6 +197,17 @@ def deleted_user_accounts_report(email:, name:, issuers:, data:)
     mail(to: email, subject: t('user_mailer.deleted_accounts_report.subject'))
   end
 
+  def account_verified(user, email_address, date_time:, app:, disavowal_token:)
+    return unless email_should_receive_nonessential_notifications?(email_address.email)
+
+    with_user_locale(user) do
+      @date = I18n.localize(date_time, format: '%B %-d, %Y')
```

Comment:
To ensure our dates are properly localized
![image](https://user-images.githubusercontent.com/1430443/117365884-947c7580-ae85-11eb-93a7-f484a83987bb.png)

Reply:
Can we add this format to [time/fr.yml](https://github.com/18F/identity-idp/blob/main/config/locales/time/fr.yml#L20) and name it?

Reply:
thanks for pointing that file out, added in 3462f4be5